### PR TITLE
Update throttling.md

### PR DESCRIPTION
**Pull request by lbrenne**
Updated Limits for Assignments Service API

### CODE REVIEW

File: concepts/throttling.md [hunk 1]
```diff
@@ -459,6 +459,18 @@ dataPolicyOperation.
...
+
+| Request Type                 | Limit per app per tenant     | Limit per tenant for all apps |
+|---------------------------|------------------------------|----------------------------|
+| Any         | 5K per 10 sec   | 15K per request per 10 sec |
+| GET MeAssignment)  | 50 per 10 sec | 150 per 10 sec |  
+
+The preceding limits apply to the following resources: 
+[educationAssignment](/graph/api/resources/educationassignment?view=graph-rest-beta)
+[educationSubmission](/graph/api/resources/educationsubmission?view=graph-rest-beta)
+[educationResourc](/graph/api/resources/educationresource?view=graph-rest-beta)
```
Comment (baywet) [inline concepts/throttling.md:472]:
```suggestion
[educationResource](/graph/api/resources/educationresource?view=graph-rest-beta)
```

File: concepts/throttling.md [hunk 2]
```diff
@@ -459,6 +459,18 @@ dataPolicyOperation.
 
 [!INCLUDE [Education rostering APIS throttling documentation](../includes/throttling-education-rostering-apis.md)]
 
+The following limits apply to requests on the assignment service beta API:
+
+| Request Type                 | Limit per app per tenant     | Limit per tenant for all apps |
+|---------------------------|------------------------------|----------------------------|
+| Any         | 5K per 10 sec   | 15K per request per 10 sec |
```
Comment (baywet) [inline concepts/throttling.md:466]:
```suggestion
| Any         | 5000 requests per 10 seconds   | 15 000 requests per 10 seconds |
```

File: concepts/throttling.md [hunk 3]
```diff
@@ -459,6 +459,18 @@ dataPolicyOperation.
 
 [!INCLUDE [Education rostering APIS throttling documentation](../includes/throttling-education-rostering-apis.md)]
 
+The following limits apply to requests on the assignment service beta API:
```
Comment (baywet) [inline concepts/throttling.md:462]: Add a section title

File: concepts/throttling.md [hunk 4]
```diff
@@ -459,6 +459,18 @@ dataPolicyOperation.
...
+
```
Comment (baywet) [inline concepts/throttling.md:463]: your changes need to be outside of the `<!-- { "blockType": "throttlinggenstart" } -->` tags as this section is generated automatically and will be overwritten.

**Review by baywet — CHANGES_REQUESTED** [review]
@Lauragra thanks for adding me as a reviewer.

@lbrenne thanks for providing that additional information. I left a few comment for you to address.

File: concepts/throttling.md [hunk 5]
```diff
@@ -459,6 +459,19 @@ dataPolicyOperation.
 
 [!INCLUDE [Education rostering APIS throttling documentation](../includes/throttling-education-rostering-apis.md)]
 
+### Assignment service limits
+The following limits apply to requests on the assignment service beta API:
+
+| Request Type                 | Limit per app per tenant     | Limit per tenant for all apps |
+|---------------------------|------------------------------|----------------------------|
+| Any         | 5000 requests per 10 seconds   | 15 000 requests per 10 seconds |
+| GET MeAssignment)  | 50 per 10 sec | 150 per 10 sec |
```
Comment (baywet) [inline concepts/throttling.md:468]:
```suggestion
| GET MeAssignment  | 50 requests per 10 seconds | 150 requests per 10 seconds |  
```
The API path doesn't look right `MeAssignement`, could it be `me/assignments` or something else?

File: concepts/throttling.md [hunk 6]
```diff
@@ -459,6 +459,19 @@ dataPolicyOperation.
 
 [!INCLUDE [Education rostering APIS throttling documentation](../includes/throttling-education-rostering-apis.md)]
 
+### Assignment service limits
+The following limits apply to requests on the assignment service beta API:
```
Comment (baywet) [inline concepts/throttling.md:463]:
```suggestion

The following limits apply to requests on the assignment service beta API:
```

**Review by baywet — CHANGES_REQUESTED** [review]
thanks for making the updates, a few more comments. And the whole section still needs to be moved outside of the auto-generation tags

**Review by davidmu1 — APPROVED** [review]
I don't have anything to add that Laura hasn't already said.

File: concepts/throttling.md [hunk 7]
```diff
@@ -454,11 +454,13 @@ dataPolicyOperation.
 
 > **Note:** The resources listed above do not return a `Retry-After` header on `429 Too Many Requests` responses.
 
+
```
Comment (baywet) [inline concepts/throttling.md:457]:
```suggestion
```

File: concepts/throttling.md [hunk 8]
```diff
@@ -454,11 +454,13 @@ dataPolicyOperation.
...
 <!-- { "blockType": "throttlinggenstart" } -->
 ### Education service limits
 
 [!INCLUDE [Education rostering APIS throttling documentation](../includes/throttling-education-rostering-apis.md)]
 
+
```
Comment (baywet) [inline concepts/throttling.md:463]:
```suggestion
```

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

File: concepts/throttling.md [hunk 9]
```diff
@@ -498,3 +498,16 @@ dataPolicyOperation.
 [!INCLUDE [Subscription services throttling documentation](../includes/throttling-subscription-services.md)]
 
 <!-- { "blockType": "throttlinggenend" } -->
+
+### Assignment service limits
+The following limits apply to requests on the assignment service beta API:
```
Comment (baywet) [inline concepts/throttling.md:503]:
```suggestion

The following limits apply to requests on the assignment service beta API:
```

File: concepts/throttling.md [hunk 10]
```diff
@@ -498,3 +498,16 @@ dataPolicyOperation.
...
+
+| Request Type                 | Limit per app per tenant     | Limit per tenant for all apps |
+|---------------------------|------------------------------|----------------------------|
+| Any         | 5000 requests per 10 seconds   | 15 000 requests per 10 seconds |
+| GET me/Assignment)  | 50 requests per 10 seconds | 150 requests per 10 seconds |
```
Comment (baywet) [inline concepts/throttling.md:508]:
```suggestion
| GET me/Assignment  | 50 requests per 10 seconds | 150 requests per 10 seconds |  
```

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

**Comment by baywet** [comment]
@lbrenne Thanks for making the changes. I've tagged the pull request to be merged.

**Pull request merged**